### DISCUSSION TITLE
Disable cache in browser

### DISCUSSION THREAD

**Discussion by kosszarva** [Salamek/chromium-kiosk#96] [General]
Dear Salamek,
Thank you for this great qiosk sw.
Is there a way to disable cache in browser via terminal?
Thank you for your answer in advance,
Levente

**Comment by Salamek** [comment]
Hi, i'm not sure what do you need exactly? You need to disable disk cache permanently? or just delete it?

Disk cache is disabled in `default` "off-the-record" profile and enabled when persistent/normal profile is used, there is currently no option to disable disk cache when using normal profile (that is not something you want to really do, it should be responsibility of displayed web page to set cache expiration correctly etc.). If you want to just delete the cache from terminal of your persistent/normal profile you can just delete whole `qiosk` cache in `/var/lib/chromium-kiosk/.cache/Salamek/qiosk/QtWebEngine/YOUR_PROFILE_NAME`

**Reply by kosszarva** [reply]
Thank you for the quick answer. 
I would like to set the "off-the-record" profile. The web developer wrote that this can help prevent a web page from getting stuck sometimes. Is there an easy way to switch to this "off-the-record" profile?

**Reply by Salamek** [reply]
off-the-record profile is a default profile used after install (`PROFILE_NAME: 'default'`)

**Reply by kosszarva** [reply]
Thank you!
